src/Console_App_Calculator/Task1.cs

```diff
@@ -0,0 +1,93 @@
+﻿// <copyright file="Task1.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace CSharp1
+{
+    /// <summary>
+    /// Class Task1 contains main method.
+    /// </summary>
+    public class Task1
+    {
+        /// <summary>
+        /// Main method recieves two integers from user
+        /// </summary>
+        /// <param name="args">reads a line</param>
+        /// <exception cref="ArgumentNullException">Throws exception when args is null</exception>
+        public static void Main(string[] args)
+        {
+            if (args is null)
+            {
+                throw new ArgumentNullException(nameof(args));
+            }
+
+            int integer1, integer2, option, next;
+            string num, n;
+            do
+            {
+                getint1:
+                Console.WriteLine("Enter integer1:");
+                num = Console.ReadLine();
+                if (string.IsNullOrEmpty(num))
+                {
+                    goto getint1;
+                }
+
+                try
+                {
+                    integer1 = Convert.ToInt32(num);
+                }
+                catch (FormatException)
+                {
+                    Console.WriteLine("Enter a valid number");
+                    goto getint1;
+                }
+
+                getint2:
+                Console.WriteLine("Enter integer2:");
```

Comment:
Since above input and this are doing a same job you can Create a method and call it.